pkg/server/event_handler_test.go

```diff
@@ -388,6 +446,16 @@ isOK: true`),
 			if diff := testutil.DiffErrString(gotErr, tc.wantErrSubstr); diff != "" {
 				t.Errorf("Process(%+v) got unexpected error substring: %v", tc.name, diff)
 			}
+
+			cmpOpts := []cmp.Option{
+				protocmp.Transform(),
+				protocmp.IgnoreFields(&v1alpha1.PmapEvent{}, "payload"),
+				cmpopts.IgnoreUnexported(v1alpha1.GitHubSource{}),
```

Comment:
Why do we need these two  IgnoreUnexported?

Reply:
Forgot to remove it. Just removed.